jobs/yoyowallet_backend_platform_engineer.html

```diff
@@ -0,0 +1,60 @@
+---
+title: Backend Platform Engineer
+company: Yoyo Wallet
+url: https://boards.greenhouse.io/yoyowallet/jobs/570082
+location: London, England
```

Comment:
You can give a more precise location if you want. London is a big place. UK Postcodes work fine. This will help people find your exact office location.